xtevent/_eventols.ado

```diff
@@ -193,6 +194,25 @@ program define _eventols, rclass
 	
 	gen byte `esample' = e(sample)
 	
+	* DiD estimate 
+	
+	if "`diffavg'"!=""{
+		unab pre : _k_eq_m*
+		unab post_p : _k_eq_p*
+		loc norma = abs(`norm')
+		loc pre : subinstr local pre "_k_eq_m`norma'" "", all

```

Comment:
@rayhuang11 Generalize for cases when the normalized coefficient is for a positive event-time.